content/Hardware Support/UNO/Update-the-firmware-of-a-Modulino-node.md

```diff
@@ -42,16 +42,13 @@ In this article:
 
 2. Connect the board to your computer and [open the Cloud Editor](https://support.arduino.cc/hc/en-us/articles/13809101080732-Open-the-Cloud-Editor).
 
-3. Go to the **Examples** tab on the left-side panel, and select **From Libraries**.
+3. Open the [FirmwareUpdater sketch](https://app.arduino.cc/sketches/examples?nav=Examples&eid=modulino_0_4_2%2Fexamples%2FUtilities%2FFirmwareUpdater&slid=modulino_0_4_2).
 
-4. Search for "Modulino" and open **Modulino > Utililities > FirmwareUpdater**.
+    > [!NOTE]
+    > Alternatively, go to **Examples > From Libraries**, search for "Modulino," and open **Modulino > Utilities > FirmwareUpdater**.
```

Comment:
I think this information should appear in the main step-by-step instructions. Also, "Open the Cloud Editor" instruction in step 2 isn't necessary when using the direct link.

How about this pattern

```
2. Connect the board to your computer

3. Do one of the following:
    * Open the FirmwareUpdater example in this browser
    * Open the Cloud Editor, go to **Examples > From Libraries**, search for "Modulino," and open **Modulino > Utilities > FirmwareUpdater**.
```

or maybe something like

```
2. Connect your board to the computer.

3. Open the FirmwareUpdater example:

   * Click [here to open it directly](https://app.arduino.cc/sketches/examples?nav=Examples&eid=modulino_0_4_2%2Fexamples%2FUtilities%2FFirmwareUpdater&slid=modulino_0_4_2), or
   * In the Cloud Editor, go to Examples > From Libraries, search for "Modulino," and select Modulino > Utilities > FirmwareUpdater.
```

I suggest highlighting the sketch link with the link-external class

Reply:
Good suggestion. [76b21e0](https://github.com/arduino/help-center-content/pull/450/commits/76b21e02f22555b8100213b45cb0192515a1a12d)